brax/physics/config.proto

```diff
@@ -113,10 +107,10 @@ message Joint {
   repeated Range angle_limit = 9;
   // Strength of the force that restores joint to within the angle limits
   // If not set, defaults to a joint-dependent fraction of `stiffness`.
-  optional float limit_strength = 10;
```

Comment:
we are using the latest version of proto3 where optional means that, upon deserialization, you can actually check for the presence of scalars.  So this is important for us to keep.